src/index.mako.html

```diff
@@ -734,6 +734,8 @@
           return decodeURIComponent((new RegExp('[?|&]' + name + '=' + '([^&;]+?)(&|#|;|$)').exec(location.search)||[,""])[1].replace(/\+/g, '%20'))||null;
         }
 
+        var wmsUrl = getParam('wms_url');
```

Comment:
I think we should treat it as api_url/api_env and add wms_urland wms_env.

Specificying a complete URL as parameter opens door for hijacking map.geo.admin.ch byu specifying a foreign wms URL altogether. (map.geo.admin.ch/?wms_url=show.funny.things/wms)

wms_url -> points to branch if specified (as api_url)
wms_env -> uses correct environment (dev int or prod)

(Maybe we should rename to wms_path and wms_staging - and api_path and api_staging)

Alternative to wms_path and wms_staging is to check with regexp that only correct domains are specified. In fact, this might be the best/most flexible solution.